src/PostgREST/DbStructure.hs

```diff
@@ -523,7 +552,10 @@ allPrimaryKeys tabs =
  where
   sql = [q|
     /*
-    -- CTE to replace information_schema.table_constraints to remove owner limit
```

Comment:
you replaced the wrong comment, it's a completely different query then the one you were working on